examples/feeds/get_feed_items_of_feed_item_set.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets all feed items of the specified feed item set.
+
+To create a new feed item set, run create_feed_item_set.py.
+To link a feed item to a feed item set, run link_feed_item_set.py.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.util import ResourceName
+
+
+def main(client, customer_id, feed_id, feed_item_set_id):
+    ga_service = client.get_service("GoogleAdsService", version="v6")
```

Comment:
Need PyDoc comment string

Reply:
Alright. Note that get_campaigns doesn't have this PyDoc.

Reply:
Yes, we haven't updated all examples to use the new conventions yet. (We did update the internal style documentation).